BIPs/2023-W31/BIP-395A.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1691004843240,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x9abeb5c10c633c53bf8f71bea4d1c4379856c2f144d66106b84c4c315a77ce51"
+  },
+  "transactions": [
+    {
+      "to": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
+      "value": "0",
+      "data": "0xdc937e1c0000000000000000000000002cc1ade245020fc5aae66ad443e1f66e01c54df10000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
```
c.decode_input("0xdc937e1c0000000000000000000000002cc1ade245020fc5aae66ad443e1f66e01c54df10000000000000000000000000000000000000000000000000000000000000000")
("delegateByType(address,uint8)", ['0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1', 0])
```

Reply:
0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1 not found in reversebook

Reply:
https://github.com/BalancerMaxis/bal_addresses/pull/96

Reply:
Adding full json